packages/core/lib/services/credential/service.js

```diff
@@ -21,6 +23,15 @@ const resolver = new MultiResolver(
   new UniversalResolver('https://uniresolver.io'),
 );
 
+let bbsPlusPresentation;
```

Comment:
why are we keeping this as state? Do we need it? If so, what happens if I generate two presentations?